Build.md

```diff
@@ -2,21 +2,23 @@
 
 # With Cmake  
 
-1. Create your build directory and go to it
+1. Create your build directory and go to it (in Kokkos Tools e.g. type `mkdir myBuild; cd myBuild`)
 
-2. Type `ccmake ..`  and change any options, including tools you want turned on  (some are by default off).  (Optional)
-
-3. Type `cmake ..` 
-
-4. Type `make` 
-
-5. Specify the generated .dylib file in the environment variable KOKKOS_TOOLS_LIBRARY when running your Kokkos-based application. 
+2. Type `cmake .. -DCMAKE_INSTALL_PREFIX=${YOUR_KOKKOS_TOOLS_INSTALL_DIR}`  and change any options, including tools you want turned on  (some are by default off).  (Optional)
 
+3. Type `make` 
 
 # With Makefile (recommended)
```

Comment:
This needs to be updated (not as part of this PR)
We do not recommend using makefiles, we will eventually get rid of them.